packages/core/src/external/commonwell/cq-bridge/coverage-enhancer-local.ts

```diff
@@ -46,6 +46,15 @@ export class CoverageEnhancerLocal extends CoverageEnhancer {
       log(`CQ orgs: ${total}, chunks: ${chunks.length}/${chunks.length + fromOrgChunkPos}`);
       log(`patients: ${patientIds.join(", ")}`);
 
+      const originalOrgs =
+        (await this.cwManagementApi.getIncludeList({ oid: orgOID }).catch(error => {
+          // intentionally ignoring the error so we can keep trying to EC
+          log(`ERROR - couldn't revert the list of CQ orgs to the original ones`, error);
+          this.capture.message("Error trying to store original CW CQ include list", {
```

Comment:
nit: `Error trying to load original CW CQ include list`?

Reply:
good catch, I updated one and forgot about the other one, one min...